source/developers/references/tools_for_maintainers.rst

```diff
@@ -0,0 +1,22 @@
+Tools for Maintainers
+#####################
+
+.. list-table::
```

Comment:
We should include Discourse and GitHub issues in this list.  I think during the kick-off we were discussing channels for maintainers to monitor/broadcast.